src/Repository/YesplanEventRepository.php

```diff
@@ -70,18 +70,20 @@ public function findOldEvents(): array
      */
     public function findNewProductionOnlineEvents(): array
     {
-        //get all events with productiononline = 1 not already created in Asana
+        // get all events with productiononline = 1 not already created in Asana
         // returns an array of event id's
+        $entityManager = $this->getEntityManager();
 
-        $conn = $this->getEntityManager()->getConnection();
-
-        $sql = '
-        SELECT y.id FROM yesplan_event y LEFT JOIN asana_event a ON y.id=a.id WHERE y.production_online = true AND  (a.created_in_new_events is null OR a.created_in_new_events = 0) AND y.profile_id = :profileIdIntern
-            ';
-        $stmt = $conn->prepare($sql);
-        $stmt->execute(['profileIdIntern' => $this->options['yesplan_intern_profile_id']]);
-
-        return $stmt->fetchAll();
+        $query = $entityManager->createQuery(
+            'SELECT y.id
+            FROM App\Entity\YesplanEvent y
+            LEFT JOIN App\Entity\AsanaEvent a WITH y.id=a.id
+            WHERE y.productionOnline = true AND  (a.createdInNewEvents is null OR a.createdInNewEvents = 0)
```

Comment:
`createdInNewEvents` is a boolean value and should not be compared to an integer. Use `… OR a.createdInNewEvents = false`.